components-loader.js

```diff
@@ -10,9 +10,13 @@ module.exports = function swingsetComponentsLoader() {
   const { pluginOptions, webpackConfig } = getOptions(this)
 
   // Resolve components glob
-  const allComponents = globby.sync(`${pluginOptions.componentsRoot}`, {
-    onlyFiles: false,
-  })
+  const allComponents = globby
+    .sync(`${pluginOptions.componentsRoot}`, {
+      onlyFiles: false,
+    })
```

Comment:
💭 [idea] could we use the `gitignore` option and/or the `ignoreFiles` option here? https://github.com/sindresorhus/globby#gitignore

Reply:
My only concern would be for users who are applying a compile-step to their components (for example, from TS -> JS). Using `.gitignore` might be too wide a net when we really just want to make sure we're not treating a nested `node_modules` folder as a component.

That being said, maybe we shift that burden to the user by removing this and just making sure that `componentsRoot` supports arrays with negated patterns: `['packages/**/*', '!packages/**/node_modules']`

Reply:
Fair point on `.gitignore`. Excluding `node_modules` by default feels reasonable though!